src/config.rs

```diff
@@ -47,29 +53,20 @@ pub(crate) fn db_config_table_creation_query() -> &'static str {
 pub(crate) fn db_announcement_table_creation_query() -> &'static str {
 	"CREATE TABLE IF NOT EXISTS channel_announcements (
 		id SERIAL PRIMARY KEY,
-		short_channel_id character varying(255) NOT NULL UNIQUE,
-		block_height integer,
-		chain_hash character varying(255),
+		short_channel_id bigint NOT NULL UNIQUE,
 		announcement_signed BYTEA,
 		seen timestamp NOT NULL DEFAULT NOW()
 	)"
 }
 
 pub(crate) fn db_channel_update_table_creation_query() -> &'static str {
+	// We'll run out of room in composite index at block 8,388,608 or in the year 2286
 	"CREATE TABLE IF NOT EXISTS channel_updates (
 		id SERIAL PRIMARY KEY,
-		composite_index character varying(255) UNIQUE,
-		chain_hash character varying(255),
-		short_channel_id character varying(255),
-		timestamp bigint,
-		channel_flags integer,
-		direction integer,
+		composite_index character(29) UNIQUE,
+		short_channel_id bigint NOT NULL,
+		direction boolean NOT NULL,
 		disable boolean,
-		cltv_expiry_delta integer,
```

Comment:
I hope all of these deletions are worth it, because we definitely won't be able to make interesting analyses about the network changes this way.

Reply:
ok, looks much better. Re: block_height, fwiw, you can derive all these properties from the blob; the value I was alluding to in my previous comment is that by having them in the db, you can run queries on the data without needing to extract each value first. But that said, I agree that block height is unlikely to ever be interesting considering seen is more useful.